lib/HLSL/DxilCondenseResources.cpp

```diff
@@ -515,7 +524,8 @@ class DxilLowerCreateHandleForLib : public ModulePass {
     if (DM.GetCBuffers().size())
       bChanged |= PatchTBuffers(DM);
 
-
+    // Assign resource binding overrides.
+    hlsl::ApplyBindingTableFromMetadata(DM);
```

Comment:
Because you use this here, you need to add a dependency to the new DxcBindingTable LLVM lib in the LLVMBuild.txt file in the lib/HLSL directory. 

The linux builds are failing because cmake generates linking commands according to library interdependencies and the linux linker is persnickety about the order they are in while other platforms don't care provided everything you need is in there somewhere.

Reply:
Thank you Greg! Worked this time!

I thought I just had to add the dependency in the target binary's CMakeLists. It didn't occur to me I had to add a dependency in LLVMBuild.txt.